packages/flutter_test/lib/src/_binding_io.dart

```diff
@@ -10,7 +10,7 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:path/path.dart' as path;
 // ignore: deprecated_member_use
```

Comment:
Looks like scaffolding.dart isn't deprecated, can we remove this ignore then? (here and elsewhere)

https://github.com/dart-lang/test/blob/master/pkgs/test_api/lib/scaffolding.dart

Reply:
It's deprecated in an older version of `test_api`, I was still getting diagnostics but maybe I just hadn't updated my local checkout to the lasted pub package versions.